src/persistence/settingsserializer.cpp

```diff
@@ -510,38 +518,37 @@ void SettingsSerializer::readIni()
             if (!groups[g].startsWith(arrayPrefix))
                 continue;
             bool ok;
-            quint64 groupArrayIndex = groups[g].mid(arrayPrefix.size()).toInt(&ok);
+            int groupArrayIndex = groups[g].mid(arrayPrefix.size()).toInt(&ok);
             if (!ok)
                 continue;
             groupsToKill.append(g);
-            //qDebug() << "Found element"<<groupArrayIndex<<"of array"<<a.name;
 
-            if (groupArrayIndex > a.size)
-                a.size = groupArrayIndex;
+            if (static_cast<quint64>(groupArrayIndex) > a.size)
+                a.size = static_cast<quint64>(groupArrayIndex);
 
             // Associate the values for this array index
             for (int vi=values.size()-1; vi>=0; vi--)
             {
                 Value& v = values[vi];
                 if (v.group != g)
                     continue;
-                groupSizes[g]--;
+                groupSizes[static_cast<size_t>(g)]--;
                 v.group = a.group;
                 v.array = ai;
-                v.arrayIndex = groupArrayIndex;
-                a.values.append(vi);
-                //qDebug() << "Found key"<<v.key<<"at index"<<groupArrayIndex<<"of array"<<a.name;
+                v.arrayIndex = static_cast<int>(groupArrayIndex);
+                a.values.append(static_cast<quint64>(vi));
             }
         }
     }
 
     // Clean up spurious array element groups
-    sort(begin(groupsToKill), end(groupsToKill), std::greater_equal<int>());
+    sort(std::begin(groupsToKill), std::end(groupsToKill),
```

Comment:
What sort is this calling if not std::sort?
I'm not super fond of gratuitous nested-name specifiers when there's no ambiguity, but if you're going to add them, why leave an unqualified name in the middle of it!?